plugins/kubernetes/app/models/kubernetes/template_filler.rb

```diff
@@ -267,6 +269,11 @@ def set_history_limit
       template[:spec][:revisionHistoryLimit] ||= 1
     end
 
+    # disable service links.
+    def set_disable_service_links
+      template[:spec][:template][:spec][:enableServiceLinks] ||= false
```

Comment:
we should set that on pod_templates for consistency, not only Deployments, use `pod_template`